### DISCUSSION TITLE
smartcrop produces bad result sometimes

### DISCUSSION THREAD

**Discussion by tony612** [libvips/libvips#2056] [Ideas]
For example, this image
![source](https://user-images.githubusercontent.com/1253659/75152572-52658900-5744-11ea-99f1-ec94664b7bc7.jpg)

will produce
![out](https://user-images.githubusercontent.com/1253659/75152059-1d0c6b80-5743-11ea-9abf-0bd8f3f3715c.jpg)

by using
```
vipsthumbnail source.jpg --size 1920x676 --smartcrop attention -o out.jpg
```

smartcrop.js can produce for this https://29a.ch/sandbox/2014/smartcrop/examples/testbed.html
<img width="1175" alt="Screen Shot 2020-02-24 at 8 24 04 PM" src="https://user-images.githubusercontent.com/1253659/75152280-a15eee80-5743-11ea-8783-728c580e0225.png">

This is another example
![source2](https://user-images.githubusercontent.com/1253659/75152522-319d3380-5744-11ea-829c-aa18c5baeedd.jpg)
produces
![out2](https://user-images.githubusercontent.com/1253659/75152539-3eba2280-5744-11ea-9a3e-99123286d774.jpg)
with the same arguments

**Comment by jcupitt** [comment]
Thanks for the examples Tony, let's tag this for the next time smartcrop is revised.
